packages/vite/src/node/build.ts

```diff
@@ -343,13 +343,13 @@ async function doBuild(
   const input = libOptions
     ? resolve(libOptions.entry)
     : typeof options.ssr === 'string'
-    ? resolve(options.ssr)
-    : options.rollupOptions?.input || resolve('index.html')
+      ? resolve(options.ssr)
+      : options.rollupOptions?.input || resolve('index.html')
```

Comment:
Seems your formatting settings are different to the projects settings
Please run the formatter